static/app/components/tooltip.tsx

```diff
@@ -79,6 +80,11 @@ type Props = DefaultProps & {
    */
   forceShow?: boolean;
 
+  /**
+   * Only display the tooltip if the content overflows
+   */
+  showOnOverflow?: boolean;
```

Comment:
This name is good, just throwing out other options if you like any of them better:  `showIfOverflow`, `showOnlyOnOverflow`

Reply:
I like that `showOnlyOnOverflow` is really clear so I'll use that, thanks!